tasks/rule_generator/src/main.rs

```diff
@@ -65,14 +80,52 @@ fn parse_test_code<'a>(expr: &'a Expression) -> Option<Cow<'a, str>> {
                         code.push_str(lit.value.as_str());
                         code.push('\n');
                     }
-                    Some(Cow::Owned(code))
+                    (Some(Cow::Owned(code)), None)
                 }
-                _ => None,
+                _ => (None, None),
             },
-            _ => None,
+            _ => (None, None),
         },
-        _ => None,
+        _ => (None, None),
+    };
+
+    test_code.map(|test_code| {
+        let option_code = option_code.map_or(Cow::Borrowed("None"), |option_code| {
+            Cow::Owned(format!("Some(serde_json::json!({option_code}))"))
+        });
+        Cow::Owned(format!(r#"("{test_code}", {option_code})"#))
+    })
+}
+
+/// Convert a javascript object literal to JSON by wrapping the property keys in double quote
+fn wrap_property_in_quotes(object: &str) -> String {
+    use regex::{Captures, Regex};
+
+    lazy_static! {
+        static ref IDENT_MATCHER: Regex = Regex::new(r"(?P<ident>[[:alpha:]]\w*)").unwrap();
+        static ref DUP_QUOTE_MATCHER: Regex =
+            Regex::new(r#"(?P<outer>"(?P<inner>"\w+")")"#).unwrap();
```

Comment:
🙈 I will have to trust this

Reply:
I agree this is awkward. Tried deserializing the text using `serde_json` but it won't work because javascript object's property names are not quoted. This function just dumbly searches for property keys and quotes them.